File: LanguAI.Frontend/src/api/models/user-view-model.ts
/* tslint:disable */
/* eslint-disable */
import { ImageViewModel } from '../models/image-view-model';
export interface UserViewModel {
  dateOfBirth?: string;
  email?: string | null;
  id?: number;
  isActive?: boolean;
  profilePicture?: ImageViewModel;
  streak?: number;
  username?: string | null;
}

File: LanguAI.Backend/LanguAI.Backend/ViewModels/User/UserViewModel.cs
using LanguAI.Backend.ViewModels.Image;

namespace LanguAI.Backend.ViewModels.User;

public class UserViewModel
{
    public int Id { get; set; }

    public string Username { get; set; }

    public DateTime DateOfBirth { get; set; }

    public string Email { get; set; }

    public bool IsActive { get; set; }

    public int Streak { get; set; }

    public ImageViewModel ProfilePicture { get; set; }
}